lib/engine/game/g_18_eus/step/loan_taker.rb

```diff
@@ -43,6 +43,19 @@ def process_payoff_loan(action)
             @round.paid_loans |= [action.entity]
             pass!
           end
+
+          def game_buttons(entity)
+            buttons = super
```

Comment:
Don't expect the return value of super to be mutable. That may not be the case.

Create an array of the new buttons and then in the last line, buttons.concat(super) ... since you know your array created in this method is mutable.

Reply:
Fixed